cli/cmd/manifest_test.go

```diff
@@ -0,0 +1,181 @@
+package cmd
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"encoding/pem"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type testServerResponse struct {
+	Status  string      `json:"status"`
+	Data    interface{} `json:"data"`
+	Message string      `json:"message,omitempty"`
+}
+
+func newTestServer(handler http.Handler) (server *httptest.Server, addr string, cert *pem.Block) {
+	s := httptest.NewTLSServer(handler)
+	return s, s.Listener.Addr().String(), &pem.Block{Type: "CERTIFICATE", Bytes: s.Certificate().Raw}
+}
+
+func TestCliManifestGet(t *testing.T) {
+	require := require.New(t)
+	s, host, cert := newTestServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.RequestURI != "/manifest" {
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		if r.Method != http.MethodGet {
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+
+		w.WriteHeader(http.StatusOK)
+		data := "Everything OK"
+
+		serverResp := testServerResponse{
+			Status: "success",
+			Data:   data,
+		}
+
+		if err := json.NewEncoder(w).Encode(serverResp); err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+		}
+	}))
+
+	defer s.Close()
+	responseFile := "test-tmp-sign.json"
```

Comment:
Ideally, you'd use afero.Fs here. But at least place this file in an ioutil.TempDir.